src/app-canvas.jsx

```diff
@@ -16,26 +16,9 @@ let AppCanvas = React.createClass({
       WebkitFontSmoothing: 'antialiased',
     };
 
-    let newChildren = React.Children.map(this.props.children, (currentChild) => {
-      if (!currentChild) { // If undefined, skip it
```

Comment:
Because AppBar itsefl takes care of it's position. There is a new prop called position.